godel_surface_detection/CMakeLists.txt

```diff
@@ -71,6 +71,11 @@ add_executable(surface_blending_service  src/services/surface_blending_service.c
 target_link_libraries(surface_blending_service ${PROJECT_NAME})
 add_dependencies(surface_blending_service godel_msgs_generate_messages_cpp)
 
+## robot_auto_blending
+add_executable(robot_auto_blending  src/nodes/robot_auto_blending.cpp)
```

Comment:
Now I get it.

Sorry for the inconvenient, I'm working on to squash them into a single
commit message.

On Wed, Mar 30, 2016 at 5:42 PM, Jorge Nicho <notifications@github.com>
wrote:

> In godel_surface_detection/CMakeLists.txt
> <https://github.com/swri-robotics/godel/pull/1#discussion_r57976432>:
>
> > @@ -71,6 +71,11 @@ add_executable(surface_blending_service  src/services/surface_blending_service.c
> >  target_link_libraries(surface_blending_service ${PROJECT_NAME})
> >  add_dependencies(surface_blending_service godel_msgs_generate_messages_cpp)
> >
> > +## robot_auto_blending
> > +add_executable(robot_auto_blending  src/nodes/robot_auto_blending.cpp)
>
> Try avoid using generic commit messages whenever possible. In general,
> commit message should be brief but still provide relevant information about
> the change that it makes.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/swri-robotics/godel/pull/1/files/f5b9165ccb2bd2909bb72c590fc6e80965b95f05..d033906e4aed3d38bda0f08dfad4a84a2f50f1dc#r57976432>
>